events/community-meeting.md

```diff
@@ -0,0 +1,73 @@
+# Kubernetes Weekly Community Meeting
+
+We have PUBLIC and RECORDED [weekly meeting](https://zoom.us/my/kubernetescommunity) every Thursday at 6pm UTC (1pm EST / 10am PST)
+ 
+Map that to your local time with this [timezone table](https://www.google.com/search?q=1800+in+utc)
+
+See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any iCal client.
+
+To be added to the calendar items, join the Google group
+[kubernetes-community-video-chat](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat) for further instructions.
+
+All meetings are archived on the [Youtube Channel](https://www.youtube.com/watch?v=onlFHICYB4Q&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ)
+
+Quick links:
+
+- [Agenda document](https://k8s.devstats.cncf.io/)
+- [Zoom meeting link](https://zoom.us/my/kubernetescommunity)
+
+## Purpose
+
+The purpose of the community meeting is the dissemination of information throughout the project. 
+Large changes in an area of Kubernetes that affect groups of people should be mentioned here. 
+
+
+## Meeting Agenda
+
+If you have a topic you'd like to present or would like to see discussed,
+please propose a specific date on the [Kubernetes Community Meeting Agenda](https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit#).
+
+General speaking the meeting is structured like this:
+
+- Introduction by Host (~3 minutes)
+- Community Demo (~10 minutes)
+- Release Updates
+  - Development Release
+  - Stable Release and point releases
+  - Older stable releases and point releases
+- Graph of the Week (~2 minutes)
+  - This is meant to bring attention to different [devstats graphs](https://k8s.devstats.cncf.io/)
+- SIG Updates
+  - Three SIGs per meeting, 10 minutes per SIG 
+- Announcements (~5 minutes)
+  - Any other community announcements should go here
+
+## Demos
+
+The first 10 minutes of a meeting is dedicated to demonstrations from the community. 
+These demos are noted at the top of the community document. 
+There is a hard stop of the demo at 10 minutes, with up to 5 more minutes for questions.
+Feel free to add your demo request to the bottom of the list, then one of the organizers will get back to you to schedule an exact date. 
+Demo submissions MUST follow the the requirements listed below. 
+
+### Requirements
+
+This meeting is the largest meeting in the community, and is attended by many people, therefore we ask that you _prepare ahead of time_ and not waste this opportunity.
```

Comment:
This meeting has a large number of attendees. Out of respect for their time, we ask that you be fully prepared for your demo. Make sure slides are clear if applicable, and the link to them is posted in the meeting agenda. Also, if you are doing a live coding demo, please make sure it has a reasonable chance of completing within the allotted time box.